errors/errors.go

```diff
@@ -89,4 +89,8 @@ var (
 	ErrUserDataNotAllowed             = errors.New("repodb: user data operations are not allowed")
 	ErrCouldNotPersistData            = errors.New("repodb: could not persist to db")
 	ErrDedupeRebuild                  = errors.New("dedupe: couldn't rebuild dedupe index")
+	ErrCosignDirNotSet                = errors.New("signatures: cosignDir is not set")
```

Comment:
Let's try not to call out specific projects for this.
Would just say ErrSignConfigDirNotSet